cmd/cloudFoundryCreateSpace.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/SAP/jenkins-library/pkg/cloudfoundry"
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+)
+
+func cloudFoundryCreateSpace(config cloudFoundryCreateSpaceOptions, telemetryData *telemetry.CustomData) {
+
+	c := command.Command{}
+	cf := cloudfoundry.CFUtils{Exec: &command.Command{}}
+
+	// reroute command output to logging framework
+	c.Stdout(log.Writer())
+	c.Stderr(log.Writer())
+
+	err := runCloudFoundryCreateSpace(&config, telemetryData, cf, &c)
+
+	if err != nil {
+		log.Entry().WithError(err).Fatal("step execution failed")
+	}
+
+}
+
+func runCloudFoundryCreateSpace(config *cloudFoundryCreateSpaceOptions, telemetryData *telemetry.CustomData, cf cloudfoundry.CFUtils, s command.ShellRunner) (err error) {
+
+	var c = cf.Exec
+
+	cfLogin := s.RunShell("/bin/bash", fmt.Sprintf("yes '' | cf login -a %s -u %s -p %s", config.CfAPIEndpoint, config.Username, config.Password))
```

Comment:
There is still no better solution than the `yes`, right? We have to talk to the cf colleagues about giving us a non-interactive option.

Reply:
perhaps, as of now, to my knowledge, this is the only way.